tests/test_client_request.py

```diff
@@ -1230,3 +1230,61 @@ def test_loose_cookies_types(loop: Any) -> None:
 def test_gen_default_accept_encoding(has_brotli: Any, expected: Any) -> None:
     with mock.patch("aiohttp.client_reqrep.HAS_BROTLI", has_brotli):
         assert _gen_default_accept_encoding() == expected
+
+
+@pytest.mark.parametrize(
+    ("trust_env", "expected_auth"),
+    (
+        pytest.param(
+            True,
+            helpers.BasicAuth("username", "pass"),
+            id="reading netrc requested",
+        ),
+        pytest.param(
+            False,
```

Comment:
It still looks like this is an entirely different test case.

Reply:
I'll split it up. I think this was one of your suggested edits initially though :)

Reply:
Split